Package.swift

```diff
@@ -16,57 +16,71 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/apple/swift-log.git", from: "1.5.3"),
-        .package(url: "https://github.com/apple/swift-nio.git", from: "2.56.0"),
+        .package(url: "https://github.com/apple/swift-nio.git", from: "2.62.0"),
     ],
     targets: [
         .target(
             name: "ConsoleKit",
             dependencies: [
                 .target(name: "ConsoleKitCommands"),
                 .target(name: "ConsoleKitTerminal"),
-            ]
+            ],
+            swiftSettings: swiftSettings
         ),
         .target(
             name: "ConsoleKitCommands",
             dependencies: [
                 .product(name: "Logging", package: "swift-log"),
                 .product(name: "NIOConcurrencyHelpers", package: "swift-nio"),
                 .target(name: "ConsoleKitTerminal"),
-            ]
+            ],
+            swiftSettings: swiftSettings
         ),
         .target(
             name: "ConsoleKitTerminal",
             dependencies: [
                 .product(name: "Logging", package: "swift-log"),
                 .product(name: "NIOConcurrencyHelpers", package: "swift-nio"),
-            ]
+            ],
+            swiftSettings: swiftSettings
         ),
         .testTarget(
             name: "ConsoleKitTests",
-            dependencies: [.target(name: "ConsoleKit")]
+            dependencies: [.target(name: "ConsoleKit")],
+            swiftSettings: swiftSettings
         ),
         .testTarget(
             name: "AsyncConsoleKitTests",
-            dependencies: [.target(name: "ConsoleKit")]
+            dependencies: [.target(name: "ConsoleKit")],
+            swiftSettings: swiftSettings
         ),
         .testTarget(
             name: "ConsoleKitPerformanceTests",
-            dependencies: [.target(name: "ConsoleKit")]
+            dependencies: [.target(name: "ConsoleKit")],
+            swiftSettings: swiftSettings
         ),
         .executableTarget(
             name: "ConsoleKitExample",
-            dependencies: [.target(name: "ConsoleKit")]
+            dependencies: [.target(name: "ConsoleKit")],
+            swiftSettings: swiftSettings
         ),
         .executableTarget(
             name: "ConsoleKitAsyncExample",
-            dependencies: [.target(name: "ConsoleKit")]
+            dependencies: [.target(name: "ConsoleKit")],
+            swiftSettings: swiftSettings
         ),
         .executableTarget(
             name: "ConsoleLoggerExample",
             dependencies: [
                 .target(name: "ConsoleKit"),
                 .product(name: "Logging", package: "swift-log"),
-            ]
+            ],
+            swiftSettings: swiftSettings
         ),
     ]
 )
+
+var swiftSettings: [SwiftSetting] { [
+    .enableUpcomingFeature("ForwardTrailingClosures"),
+    .enableUpcomingFeature("ConciseMagicFile"),
+] }
```

Comment:
Mostly curiosity, is there a reason we are opting for a computed property here rather than a constant?

Reply:
Because it lets me put the settings at the bottom of the file instead of the top (and I must admit it was @MahdiBM who came up with that).